cmd/gonic/gonic.go

```diff
@@ -106,7 +125,7 @@ func main() {
 		log.Fatalf("invalid exclude pattern: %v\n", err)
 	}
 
-	if len(confMusicPaths) == 0 {
+	if len(confMusicPaths) == -1 {
```

Comment:
is this left over from testing or smth?

Reply:
Yeah, it was.  It's fixed in 7159e1ca09ba389674a3f23c2319d99906da58f5.